lintro/ai/json_response.py

```
@@ -70,3 +73,66 @@ def load_json_object(*, content: str) -> dict[str, Any]:
+
+    Returns:
+        Parsed fix payload as an object or array.
+
+    Raises:
+        ValueError: When JSON is invalid.
+    """
+    json_text = strip_json_fences(content=content)
+    try:
+        payload = json.loads(json_text)
+    except json.JSONDecodeError as exc:
+        raise ValueError(f"Invalid JSON: {exc}") from exc
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)